src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discountLarge = 50;
+  const discountSmall = 20;
+
+  if (days >= 7) {
+    return ((days * price) - discountLarge);
+  }
+
+  if (days >= 3 && days <= 6) {
+    return ((days * price) - discountSmall);
+  } else {
```

Comment:
You don't have any sense to use `else` block here. You can create general `return` after all `if` blocks.

```js
if (someCondition) {
  return some
}

if (anotheCondition) {
  return another
}

return default
```

Reply:
Thank you! Done ✓